Bio/bgzf.py

```diff
@@ -404,7 +404,7 @@ def BgzfBlocks(handle):
 
 
 def _load_bgzf_block(handle, text_mode=False):
-    """Internal function to load the next BGZF function (PRIVATE)."""
+    """Load the next BGZF function (INTERNAL PRIVATE)."""
```

Comment:
We've a convention (which may not be documented anywhere?) using ``(PRIVATE)`` at the end of docstrings, and also ``(DEPRECATED)`` and ``(OBSOLETE)`` as per http://biopython.org/wiki/Deprecation_policy but I don't think we've ever used ``INTERNAL`` like this.

How about:

```python
"""Load the next BGZF function (PRIVATE)."""
```